### ISSUE TITLE
Blind revision is not so blind

### ISSUE THREAD

**Issue by olea** [openSUSE/osem#2025]
**I'm submitting a ..**
- [ x] Bug Report
- [ ] Feature Request

**Current behavior:**

At first glance the blind revision looks pretty fine:

![captura de pantalla de 2018-04-03 12-16-36](https://user-images.githubusercontent.com/141267/38243812-fe7e2c34-3738-11e8-9efa-be3336a34f58.png)

But when reading the proposal, if the author added personal information to biography it brokes anonimity:

![captura de pantalla de 2018-04-03 12-18-31](https://user-images.githubusercontent.com/141267/38243900-42cb98a4-3739-11e8-85b3-7b32deb94e76.png)

**Expected correct behavior:**

Somehow, to keep anonymized the whole process. If not possible then to disable the (nice to me) blind revision feature, I guess.

**Comment by differentreality** [comment]
So the biographies field should be hiding the name of the speaker (when blind voting is on).

**Comment by altsalt** [comment]
This seems like an oversight. Blind reviews are designed such that the reviewers have no information about the proposal.

Blind voting is a separate, but also important feature, such that the reviewers don't know how their fellow reviewers voted.

**Comment by bear454** [comment]
> This seems like an oversight. Blind reviews are designed such that the reviewers have no information about the proposal.
> 
> Blind voting is a separate, but also important feature, such that the reviewers don't know how their fellow reviewers voted.

Agreed; this issue needs to be renovated as a new feature request, to add blind reviews, in addition to the blind voting.

**Comment by Ana06** [comment]
Related to https://github.com/openSUSE/osem/issues/2335
